src/mumble/AudioOutput.ui

```diff
@@ -551,7 +551,7 @@
       <item row="4" column="2">
        <widget class="QSlider" name="qsMaxDistVolume">
         <property name="toolTip">
-         <string>Factor for sound volume decrease</string>
+         <string>The minimum volume a player's audio will fade out to with increasing distance. Set to 0% for a realistic experience.</string>
```

Comment:
I think we should be more specific here and explain what makes it realistic. Namely, Set to 0 % for it to fade into complete silence for a realistic maximum hearing distance.

Reply:
Sounds good :+1: